examples/client.go

```diff
@@ -88,7 +88,7 @@ func customHTTPClientUsage() error {
 			client: http.Client{Timeout: 25 * time.Millisecond},
 		}),
 		httpclient.WithRetryCount(2),
-		httpclient.WithRetrier(heimdall.NewRetrier(heimdall.NewConstantBackoff(10, 5))),
+		httpclient.WithRetrier(heimdall.NewRetrier(heimdall.NewConstantBackoff(10, 50*time.Millisecond))),
```

Comment:
What about the first argument ? That is also `time.Duration`, can you fix that plz ?

Reply:
Should be fixed now